src/MatrixEquations.jl

```diff
@@ -19,7 +20,7 @@ export lanv2, ladiv, lag2, lacn2!
 export lyapc, lyapd, lyapcs!, lyapds! 
 export plyapc, plyaps, plyapcs!, plyapd, plyapds!  
 export arec, ared, garec, gared
-export sylvc, sylvd, gsylv, sylvcs!, sylvds!, gsylvs!, gsylvs1!
```

Comment:
This didn't exist anywhere in the package.

Reply:
I mean `gsylvs1!`.

Reply:
It's OK. Probably it is reminiscent from a test version. Thanks.